packages/truffle-codec/lib/decode/abi.ts

```diff
@@ -259,11 +303,18 @@ function* decodeAbiStructByPosition(dataType: Types.StructType, location: AbiLoc
   const typeId = dataType.id;
   const structAllocation = allocations[typeId];
   if(!structAllocation) {
-    let error = new Errors.UserDefinedTypeNotFoundError(dataType);
+    let error = {
+      kind: "UserDefinedTypeNotFoundError" as "UserDefinedTypeNotFoundError",
```

Comment:
that's what it is!

Reply:
Yeah -- for some reason you need to do that for TypeScript to assign it a string constant type instead of just the `string` type.  It's annoying.